docs-src/python/interrupt-a-workflow-execution.md

```diff
@@ -0,0 +1,49 @@
+---
+id: interrupt-a-workflow-execution
+title: How to interrupt a Workflow Execution
+description:
+sidebar_label: Interrupt a Workflow Execution
+tags:
+  - guide-context
+---
+
+You can interrupt a Workflow Execution in one of the following ways:
+
+- [Cancel](#cancel)
+- [Terminate](#terminate)
+
+The following are the main differences between canceling and terminating a Workflow in Temporal:
+
+##### Cancel
+
+Canceling a Workflow provides a graceful way to stop Workflow Execution.
+This action resembles sending a `SIGTERM` to a process.
+
+- The system records a `WorkflowExecutionCancelRequested` event in the Workflow History.
+- The Workflow code can handle the cancelation and execute any cleanup logic.
+- A Workflow Task gets scheduled to process the cancelation.
```

Comment:
```suggestion
- A Workflow Task gets scheduled to process the cancelation.
- The Workflow code can handle the cancelation and execute any cleanup logic.
```

Reply:
Worth mentioning that the workflow may choose *not* to be cancelled.